basilisk/multiprocessing_worker.py

```
@@ -0,0 +1,115 @@
+	"""Run a task in a separate process.
+
+	Args:
+		task: The task to run
+		result_queue: The queue to store the task result
+		cancel_flag: The flag to indicate if the task should be cancelled
+		*args: The task arguments
+		**kwargs: The task keyword arguments
+	"""
+
+
+import traceback
```

Ruff (0.11.9)

Module level import not at top of file

(E402)